.github/actions/mssql/action.yml

```diff
@@ -0,0 +1,52 @@
+# This GitHub Action is designed to deploy the latest data migration to an MSSQL database on an AMI.
+
+name: 'Deploy MSSQL'
```

Comment:
Is it worth calling this MSSQL Migrations? This makes it sound like we are deploying a database not just running schema based migrations